src/Components/Movies.js

```diff
@@ -0,0 +1,69 @@
+import axios from 'axios'
+import React, { Component } from 'react'
+
+export class Movies extends Component {
+    constructor(){
+        super()
+        this.state={
+            movies: [],
+            chosenTitle: '',
+            releaseDate: '',
+            description: ''
+        }
+    }
+
+    componentDidMount(){
+        this.getMovies()
+    }
+
+    getMovies = async () => {
+        const {data} = await axios.get(`https://ghibliapi.herokuapp.com/films`)
+        this.setState({
+            movies: data,
+        })
+
+    }
+
+    handleChange = (e) => {
+
+        //condition used for if the blank option is selected
+        if (e.target.value) {
+            let movie = this.state.movies.filter(movie => movie.id === e.target.value)
+            this.setState({
+                chosenTitle: movie[0].title,
+                releaseDate: movie[0].release_date,
+                description: movie[0].description,
+            })
+        } else {
+            this.setState({
+                chosenTitle: '',
+                releaseDate: '',
+                description: '',
+            })
+        }
+    }
+
+    render() {
+        //const {movies, chosenTitle, releaseDate, description, failedApiCall} = this.state
+        const options = this.state.movies.map((movie,i) => <option key={i} value={movie.id}>{movie.title}</option>)
+        return (
+            <div>
+                <h1>Select a Movie</h1>
+                <select onChange={this.handleChange} name='' id=''>
```

Comment:
This should be a controlled component! i.e. you should pass a value prop so that react can control which option the user sees as the selected option.